docs/run_test/quick_run.md

```diff
@@ -1,66 +1,40 @@
-# Run LISA
+# Getting started with Azure
 
-- [Quick start](#quick-start)
-  - [Run in Azure](#run-in-azure)
-  - [Run in Ready computers](#run-in-ready-computers)
-- [Run Microsoft tests](#run-microsoft-tests)
-- [Run legacy LISAv2 test cases](#run-legacy-lisav2-test-cases)
+In this document we will guide you through the test procedure using a powerful
```

Comment:
It needs to unify the style, either to third person or second person, both fine to me.